src/expressServer.ts

```diff
@@ -362,14 +362,30 @@ export async function sendToSlack(text: string, slackChannel?: string, shouldThr
     return;
   }
   const slackWebhooks = JSON.parse(slackWebhooksString ?? "{}");
-  // get the webhook url from the channel name as the key
-  const slackWebhookUrl = slackWebhooks[slackChannel ?? "default"];
+
+  // use "default" channel if no slackChannel is provided
+  const channel = slackChannel ?? "default";
```

Comment:
Will you please comment this is our ops channel

Reply:
I generally avoid putting in code comments specific to our project that don't translate to all projects here in ferns-api directly, since anyone could use this library and it would not apply/be confusing. Open minded here if the team thinks otherwise! Would be a good thing to get a team consensus on. Going to move this convo and the general convo about code comments over to our team slack channel and circle back!

Reply:
that makes total sence i was thinking we were still in flourish repo